scim/data_user.go

```diff
@@ -67,6 +67,10 @@ func DataSourceUser() common.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"active": {
```

Comment:
can we switch to GoSDK & `common.WorkspaceDataWithParams` for this data source?

Reply:
I think the issue is that this data source is being used for both the workspace- and account-level. Resources that are being used at both levels cannot use `common.WorkspaceDataWithParams`, right?